tests/Fixtures/StubMutator.php

```diff
@@ -8,7 +8,7 @@
 
 class StubMutator extends Mutator
 {
-    public function mutate(Node $node)
+    public function mutate(Node $node): \Generator
```

Comment:
This can't be called in the current form. [Example.](https://3v4l.org/A3I6V)
```
TypeError: Return value of StubMutator::mutate() must be an instance of Generator, none returned
```
So you see generators are tricky.

Reply:
I would argue it's more of an example of forgetting to refactor a specific class rather than dealing with the generator trickiness here though